packages/quantic/cypress/integration/standalone-search-box/standalone-search-box-selectors.ts

```diff
@@ -0,0 +1,26 @@
+import {ComponentSelector, CypressSelector} from '../common-selectors';
+
+export const standaloneSearchBoxComponent = 'c-quantic-standalone-search-box';
+
+export interface StandaloneSearchBoxSelector extends ComponentSelector {
+  input: () => CypressSelector;
+  suggestionList: () => CypressSelector;
+  closeIcon: () => CypressSelector;
+  searchIcon: () => CypressSelector;
+  searchButton: () => CypressSelector;
+}
+
+export const StandaloneSearchBoxSelectors: StandaloneSearchBoxSelector = {
+  get: () => cy.get(standaloneSearchBoxComponent),
+  input: () => StandaloneSearchBoxSelectors.get().find('input[type="search"]'),
+  suggestionList: () =>
+    StandaloneSearchBoxSelectors.get().find(
+      'c-quantic-search-box-suggestions-list li'
+    ),
+  closeIcon: () =>
```

Comment:
maybe here would've been a better place to put my clearButton idea 😅

Reply:
good catch